SimPEG/Regularization.py

```diff
@@ -126,98 +413,58 @@ class Tikhonov(BaseRegularization):
     alpha_zz = Utils.dependentProperty('_alpha_zz', 0.0, ['_W', '_Wzz'], "Weight for the second derivative in the z direction")
 
     def __init__(self, mesh, mapping=None, indActive = None, **kwargs):
-        BaseRegularization.__init__(self, mesh, mapping=mapping, **kwargs)
-        self.indActive = indActive
+        BaseRegularization.__init__(self, mesh, mapping=mapping, indActive=indActive, **kwargs)
 
     @property
     def Ws(self):
```

Comment:
I like Wsmall
On Fri, Mar 25, 2016 at 10:22 PM Lindsey <notifications@github.com> wrote:

> In SimPEG/Regularization.py
> <https://github.com/simpeg/simpeg/pull/254#discussion_r57509570>:
>
> > @@ -126,98 +413,58 @@ class Tikhonov(BaseRegularization):
> >      alpha_zz = Utils.dependentProperty('_alpha_zz', 0.0, ['_W', '_Wzz'], "Weight for the second derivative in the z direction")
> >
> >      def __init__(self, mesh, mapping=None, indActive = None, **kwargs):
> > -        BaseRegularization.__init__(self, mesh, mapping=mapping, **kwargs)
> > -        self.indActive = indActive
> > +        BaseRegularization.__init__(self, mesh, mapping=mapping, indActive=indActive, **kwargs)
> >
> >      @property
> >      def Ws(self):
>
> would ppl be game if we renamed this to Wsmall?
>
> (s could mean small or smooth, so it is ambiguous...)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/simpeg/simpeg/pull/254/files/f92ff1301db92f5dda32125177281064ec59c17f#r57509570>
>